src/dune_rules/melange/melange_stanzas.ml

```diff
@@ -110,6 +112,22 @@ module Emit = struct
        and+ allow_overlapping_dependencies =
          field_b "allow_overlapping_dependencies"
        and+ modules = Stanza_common.Modules_settings.decode in
+       let () =
+         match alias with
+         | None -> ()
+         | Some a -> (
```

Comment:
```suggestion
         | Some (loc, a) -> (
```

Reply:
@jchavarri I think this comment needs to be addressed?

Reply:
It’s actually OK, we discussed it in https://github.com/ocaml/dune/pull/7327#discussion_r1141161355

Reply:
Yes, I understood this comment as part of the same change as the other one. @rgrinberg I decided to leave it as is as it was not a blocker for Antonio, but I can change it if you think it is.